node-bootstrapper/const.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+// Some options are intentionally non-configurable to avoid customization by users
+// it will help us to avoid introducing any breaking changes in the future.
+const (
+	LogFile               = "/var/log/azure/node-bootstrapper.log"
+	BootstrapService      = "bootstrap.service"
+	ProvisionJSONFilePath = "/var/log/azure/aks/provision.json"
```

Comment:
nit: consistent naming, LogFile / ProvisionJSONFile or LogFilePath / ProvisionJSONFilePath

question: what is provision.json? whats the purpose of the file?

Reply:
created here: https://github.com/Azure/AgentBaker/blob/master/parts/linux/cloud-init/artifacts/cse_start.sh#L45

which contains provisioning completion information